lib/scripts/mtw.js

```diff
@@ -292,6 +301,36 @@ export class ContentScript {
   }
 
   /**
+   * Translate one word in each sentence for a  paragraph.
+   * @param {Object} paragraph - Paragraph nodeType
+   * @param {Object} filteredTMap - filtered translation map
+   * @param {Object} iMap - HTML element for each translated word
+   */
+  translateOneWord(paragraph, filteredTMap, iMap) {
+    for (let i in paragraph.childNodes) {
+      if (paragraph.childNodes[i].nodeType === 3) {
+        if(!/^\s*$/.test(paragraph.childNodes[i].textContent)){
+          let sentences = paragraph.childNodes[i].textContent.split('.');
+          for (let j in sentences) {
+            let words = sentences[j].split(' ');
+            words = _.shuffle(words);
+            for (let k in words) {
+              if (filteredTMap[words[k]]) {
```

Comment:
Ah... I have just seen that, because of these lines, your feature is actually a "translate **at most** one word per sentence" feature, instead of "**exactly** one word per sentence". This should be made clearer to the user. And in this case, maybe a global option makes sense.